programs/cleos/main.cpp

```diff
@@ -1057,16 +1057,16 @@ struct set_action_permission_subcommand {
       auto permissions = actionRoot->add_subcommand("permission", localized("Set parameters dealing with account permissions"));
       permissions->add_option("account", accountStr, localized("The account to set/delete a permission authority for"))->required();
       permissions->add_option("code", codeStr, localized("The account that owns the code for the action"))->required();
-      permissions->add_option("type", typeStr, localized("The type of the action"))->required();
+      permissions->add_option("type", typeStr, localized("The type of the action [use ALL for all types]"))->required();
```

Comment:
do we think this should change `type` to `action` here? `type` is pretty weird and atypical for our modern terminology I think

Reply:
I found `type` very odd as well, @huangminghuang actually asked me the same question. I agree `action` would be much better name. Didn't know if we want to update that.

Reply:
maybe it makes sense to keep the argument name as "type", since that's the name in the json that is sent? :man_shrugging: I'm fine either way with the new description